company_website/templates/common_partials/head.haml

```diff
@@ -9,6 +9,14 @@
   })(window,document,'script','dataLayer','GTM-TCWZK7N');
 / End Google Tag Manager
 
+/ Global site tag (gtag.js) - Google Ads: 607460288
+%script{:async => "", :src => "https://www.googletagmanager.com/gtag/js?id={{ google_ads_conversion_id }}"}
+:javascript
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){ dataLayer.push(arguments); }
+  gtag('js', new Date());
+  gtag('config', '{{ google_ads_conversion_id }}');
```

Comment:
I saw that you delete `gtag_setup.js` file and you said that it should be in header. It can be in header but it should never be just javascript code in haml/html file. Please move it to file again. I know that you can here init variable with `google_ads_conversion_id ` setting and use it in file and please try to do this.

Reply:
Done.